plugins/UsersManager/API.php

```diff
@@ -1379,4 +1386,33 @@ private function getRoleAndCapabilitiesFromAccess($access)
         }
         return [$roles, $capabilities];
     }
+
+    private function sendEmailChangedEmail($user, $newEmail)
+    {
+        $userAgent = isset($_SERVER['HTTP_USER_AGENT']) ? $_SERVER['HTTP_USER_AGENT'] : '';
+
+        $uaParser = new DeviceDetector($userAgent);
+        $uaParser->parse();
+
+        $view = new View('@UsersManager/_emailChangedEmail.twig');
+        $view->accountName = Common::sanitizeInputValue($user['login']);
+        $view->newEmail = Common::sanitizeInputValue($newEmail);
+        $view->deviceName = $uaParser->getDeviceName();
```

Comment:
Might it maybe make sense to add some more details about the device if available? `getDeviceName` only give `desktop, smartphone, ...` Maybe we could if available also add `getBrand()` and `getModel()`. So we could print something like `smartphone (Sony Xperia Z5)` or only `dekstop` if not specified

Reply:
👍 will do